test_sets/External_UCS-related_params.txt

```diff
@@ -0,0 +1,22 @@
+./test_scripts/Polices/External_UCS/001_ATF_P_TC_PreloadedPT_without_external_consent_status_groups_struct.lua Check_that_SDL_starts_successfully_in_case_external_consent_status_groups_struct_is_omitted_in_PreloadedPT
```

Comment:
@istoimenova 
1) Regarding missing functional req-t appropriate question was raised and assigned to BA
2) Mentioned link is opened successfully for me